src/main/java/teammates/ui/controller/InstructorCourseEnrollPageData.java

```diff
@@ -6,14 +6,24 @@
  * PageData: this is page data for 'Enroll' page for a course of an instructor
  */
 public class InstructorCourseEnrollPageData extends PageData {
+    private String courseId;
+    private String enrollStudents;
     
-    public InstructorCourseEnrollPageData(AccountAttributes account) {
+    public InstructorCourseEnrollPageData(AccountAttributes account, String courseId, String enrollStudents) {
         super(account);
-        enrollStudents = "";
+        this.courseId =  PageData.sanitizeForHtml(courseId);
+        this.enrollStudents = PageData.sanitizeForHtml(enrollStudents);
```

Comment:
Ah, this extends from `PageData`, so there's no need for the static reference to it.
Just do a `this.courseId = sanitizeForHtml(courseId);`